packages/web-pkg/src/components/AppTemplates/AppWrapper.vue

```diff
@@ -479,9 +482,16 @@ export default defineComponent({
     }
 
     const menuItemsContext = computed(() => {
-      return [...unref(openWithAppActions), ...unref(fileActionsSave)].filter((item) =>
-        item.isVisible(unref(actionOptions))
-      )
+      return [
+        ...unref(openWithAppActions),
+        ...unref(fileActionsSave),
+        ...unref(saveAsActions).map((action) => {
+          return {
+            ...action,
+            isVisible: (args: FileActionOptions) => isEditor.value && action.isVisible(args)
```

Comment:
You could also pass the `isEditor` computed to the `useFileActionsSaveAs` composable and check it there. Does it make sense, as in: does this action generally only make sense for editors? Or is this specific logic bound to this component and might be different elsewhere?

Reply:
As for now having the save as is only implemented in the top bar (app). BUT we could have it later somewhere else like in the context menu for every other file 🤷‍♀️